webapp/src/cloudsync.ts

```diff
@@ -501,7 +503,7 @@ export function refreshToken() {
 }
 
 export function syncAsync(): Promise<void> {
-    return Promise.all([githubSyncAsync(), cloudSyncAsync()])
+    return Promise.all([githubSyncAsync(), cloud.syncAsync()])
```

Comment:
note important change: instead of local cloudSyncAsync function, we're pointing to new cloud.syncAsync